### DISCUSSION TITLE
Highlights if any user writes

### DISCUSSION THREAD

**Discussion by Finetnow** [Chatterino/chatterino2#3563] [Q&A] [answered]
Hi, I apologize in advance if this question is as dumb as possible. Can I make it so that I am notified if at least someone has written in the chat? (not a specific person, but absolutely random)

**Comment by Felanbird** [comment, marked as answer]
Yes and no.

You can add a highlight that's just `.+` with regex enabled and every message sent will be highlighted/notified, but this is basically only a workaround if you have a very small pool of channels open <5 imo, otherwise every message in a large/fast channel will highlight/notify, which will get annoying fast.

**Reply by Finetnow** [reply]
Tyvm!

**Comment by seanbperiod** [comment]
Would it be possible to add functionality so that "new activity" or message from a specific channel will only receive the alert?
